clinica/cmdline.py

```diff
@@ -69,5 +73,13 @@ def cli(verbosity: int = 0) -> None:
 cli.add_command(run_cli)
 
 
+def main() -> None:
+    try:
+        cli()
+    except ClinicaException as e:
+        cprint(msg=e, lvl="error")
```

Comment:
lvl=error prints by default, doesn't it? I'm still getting the full traceback when running into a Clinica Exception

Reply:
The cprint default level is `info`

Reply:
You'll need to do a `pip install -e .` again since the entry point has changed